src/sas/qtgui/Plotting/SlicerParameters.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, parent=None,
         self.active_plots = active_plots
 
         # Initially, Apply is disabled
-        self.cmdApply.setEnabled(False)
+        #self.cmdApply.setEnabled(False)
```

Comment:
@butlerpd Here's one example of the command commented out related to the batch slicer tab. The batch slicing tab was removed in the .ui file.